packages/terra-text/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Changelog
 Unreleased
 ----------
 
+### Removed
```

Comment:
Changed classnames here too

Reply:
I talked to Brett about this and he said I don't need to log class name changes because it is trackable through this https://github.com/cerner/terra-core/issues/812

Reply:
Im fine with no bullet point for the class name changes, I'd consider it an implementation detail.